dbt/models/churnado_subscriptions.sql

```diff
@@ -0,0 +1,41 @@
+select
+  s.id as subscription_id
+  , s.customer_id
+  , s.plan_id
+  , s.simplified_plan_id
+  , s.canceled_at
+  , s.created_at
+  , s.quantity
+  , s.trial_start_at
+  , s.trial_end_at
+  , u.id as user_id
+  , datediff(day, s.trial_start_at, s.trial_end_at) as trial_length
+  , case -- if subscription was active on Dec 31 and was created two months before, we can use it in the training set
+      when s.created_at < '2017-11-01' and (s.canceled_at is null or s.canceled_at > '2017-12-31')
+      then true
+      else false
+    end as is_training
+  , count(distinct case when c.captured and c.paid and c.refunded = false then c.id else null end) as successful_charges
```

Comment:
Could these `c.*` dimensions be moved to a `churnado_charges.sql` model? This way we keep each one isolated

Reply:
Sorry, I'm not sure I understand completely. Would you mind explaining why we'd do that again? 😄

I'm only wondering because I don't know of a case in which we'd want subscriptions without charges or charges without subscriptions. Would it be because it's dependent on the training date?